### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ververnik**
Want to edit with second try

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 For testing GitHub
 
 first try to edit
+
```
Comment (salabaeva) [inline README.md:5]: nothing to review

File: README.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
...
+second try to edit
```
Comment (salabaeva) [inline README.md:6]: ok

**Comment by salabaeva** [comment]
Ok

File: README.md (hunk 1 above)
Comment (Ververnik) [reply README.md:5]: Ok!

**Pull request merged**